.circleci/config.yml

```diff
@@ -1,4 +1,21 @@
 jobs:
+  run-pre-commit:
+    docker:
+      - image: cimg/python:3.9.1
+    steps:
+      - checkout
+      - run: pip3 install pre-commit
+      - run: pre-commit run --all-files --verbose
+
+  # TODO: move this check into pre-commit
+  changelog-url-test:
+    docker:
+      - image: cimg/python:3.9.1
+    steps:
+      - checkout
+      - run: pip3 install urlchecker
+      - run: cd treeherder/changelog && urlchecker check --file-types .json .
```

Comment:
Is the urlchecker validating the json file? I'd rather extend any sort of validating or linting to all .json files rather than one specific file (and ideally to toml and yml files too). I'm thinking this might be better done as part of the linter checks.

Reply:
Yes, only the json files in the changelog subdirectory. I pushed an earlier commit with a bad URL and wanted to make sure it wouldn't happen again.

I'm happy to extend it to everything in the repo, but didn't want to decide that myself.

By linter, you mean eslint?

Reply:
yes, eslint (or an eslint plugin) - thank you!

Reply:
ESLint seems focused on JS files. It didn't like the syntax for repositories.json or the other existing JSON files (I also couldn't find a link checker plugin).

modified `yarn lint` command to add JSON:
``` bash
$ node ./node_modules/eslint/bin/eslint.js --report-unused-disable-directives --max-warnings 0 --format codeframe --ext js,jsx,json  treeherder/changelog/repositories.json 
error: Parsing error: Missing semicolon

  1 | {
> 2 |   "repositories": [
    |                 ^
  3 |     {
  4 |       "metadata": {
  5 |         "description": "Tool for creating Windows cloud", at treeherder/changelog/repositories.json:2:17:
  1 | {
> 2 |   "repositories": [
    |                 ^
  3 |     {
  4 |       "metadata": {
  5 |         "description": "Tool for creating Windows cloud",


1 error found.
```

Should I just abandon the check, create an urlchecker pre-commit hook (should be easy), or do you have any other ideas? Thanks. :)

Reply:
Ah, darn. The urlchecker pre-commit hook sounds good.